Update TypeScript.md (#4126)
fastify/fastify: docs/Reference/TypeScript.md

  [npm-check](https://www.npmjs.com/package/npm-check) to verify plugin
  dependencies are being used somewhere in your project.

> Note that using `require` won't load the type definitions properly and may

Reviewer: ```suggestion
Note that using `require` will not load the type definitions properly and may
```

Reference docs are formal, so contractions shouldn't be used.

cause type errors.

> TypeScript can only identify the types that are directly imported into code
> which means that you can use require inline with import on top, for example:

Reviewer: ```suggestion
TypeScript can only identify the types that are directly imported into code,
which means that you can use require inline with import on top. For example:
```

```typescript
import 'plugin' // here will trigger the type augmentation.

fastify.register(require('plugin'))
```

```typescript
import plugin from 'plugin' //  here will trigger the type augmentation.

fastify.register(plugin)
```

Or even explicit config on tsconfig

> ```json

Reviewer: ```suggestion
```jsonc
```
